shared/contracts/BaseTemplate.sol

```diff
@@ -152,21 +150,26 @@ contract BaseTemplate is APMNamehash, IsContract {
 
     /* VOTING */
 
+    function _installVotingApp(Kernel _dao, MiniMeToken _token, uint64[3] _votingSettings) internal returns (Voting) {
+        return _installVotingApp(_dao, _token, _votingSettings[0], _votingSettings[1], _votingSettings[2]);
```

Comment:
Also, previously we were checking that the `_votingSettings` length be correct with a require statement and throwing something like "ERROR_BAD_VOTE_SETTINGS" when it was incorrect. Was this not needed?

Reply:
Note that we are already limiting that by `uint64[3]`. However, we are still catching that from the templates side